jstp/src/main/java/com/metarhia/jstp/connection/Connection.java

```diff
@@ -638,14 +763,29 @@ public void setClientMethodNames(String interfaceName, List<String> names) {
     clientMethodNames.put(interfaceName, methods);
   }
 
+  /**
+   * Checks if socket is connected
+   *
+   * @return is socket connected
+   */
   public boolean isConnected() {
     return state == ConnectionState.STATE_CONNECTED;
   }
 
+  /**
+   * Checks if connection is closed
+   *
+   * @return is connection closed
+   */
   public boolean isClosed() {
     return state == ConnectionState.STATE_CLOSED;
   }
 
+  /**
```

Comment:
I think we should avoid such comments, those are simple getter methods, we should only comment to elaborate the meaning of the value and provide better description, or in case getter or setter is not a simple return or assign statement.